classes/Search.php

```diff
@@ -128,7 +128,7 @@ class SearchCore
     public const PS_SEARCH_ORDINATE_MAX = -1;
     public const PS_SEARCH_ABSCISSA_MIN = 0.5;
     public const PS_SEARCH_ABSCISSA_MAX = 2;
-    public const PS_DISTANCE_MAX = 8;
+    public const PS_DISTANCE_MAX = 5;
```

Comment:
this contant is no more in use

Reply:
I left it there just for backward compatibility purpose, we can delete later. You know, if someone had overrides for example or just used it from some external module.